coral-component-datepicker/src/scripts/Datepicker.js

```diff
@@ -511,10 +511,9 @@ const Datepicker = Decorator(class extends BaseFormField(BaseComponent(HTMLEleme
   set required(value) {
     this._required = transform.booleanAttr(value);
     this._reflectAttribute('required', this._required);
-
-    this._elements.toggle.classList.toggle('is-invalid', this._required);
-
-    this._elements.input.required = this._required;
+    // this._elements.toggle.classList.toggle('is-invalid', this._required);
```

Comment:
@hparwani2 Is there any particular reason for removal of this class. This is a public class and can be used by teans.

Reply:
While loading the component, I've observed that the toggle button (which contains the picker) also appears in red. This happens because the 'is-invalid' attribute is toggled to 'true' when the 'required' property is set to 'true'. Ideally, this class should only be applied when the input is either invalid or when the form is submitted. This behavior is currently being managed by the 'invalid' property setter.